z390test/src/test/groovy/org/z390/test/RunAsmTests.groovy

```diff
@@ -61,6 +61,32 @@ class RunAsmTests extends z390Test {
         assert rc == 0
     }
     @Test
+    void test_TESTFPC1() {
+        env.put('SYSOUT', basePath('tests', 'TESTFPC1.TST'))
+        int rc = this.asmlg(basePath("tests", "TESTFPC1"), *options, "SYSOBJ(${basePath("linklib")})", 'optable(z390) notrace')
```

Comment:
I assume the SYSOBJ is for any *.OBJ that are included in the resulting load module. However, it also results in TESTFPC1.OBJ being put in linklib.

The following change should put TESTFPC1.OBJ and a built TESTFPC1.390 in the tests directory.

Replace "SYSOBJ(${basePath("linklib")})" with "SYSOBJ(+${basePath("linklib")})"

The "+" will add linklib after tests. TESTFPC1.OBJ will be put in tests in the assembly step. Any referenced external modules in linklib will be found and included in the link step.

Same comment applies to the TESTFPC2 test.